mkdocs.yml

```diff
@@ -299,8 +299,12 @@ extra:
     version:
         provider: mike
     analytics:
-        provider: google
-        property: G-3M31G9B0QJ
+        provider: custom
+        property: !ENV [AMPLITUDE_API_KEY, "8ab9ae57b486baa5d4ab626559fd8106"]
+        url: !ENV [AMPLITUDE_URL, "https://api2.amplitude.com/2/httpapi"]
```

Comment:
This should be able to use the Amplitude proxy? Maybe needs to be copied into the docs though

Reply:
Set via environment variable in Netlify 👍🏻 for local testing you need the full url